plugin/trino-delta-lake/README.md

```diff
@@ -0,0 +1,191 @@
+#Delta Lake Connector Developer Notes
+
+The Delta Lake connector can be used to interact with [Delta Lake](https://delta.io/) tables.
+
+Trino has product tests in place for testing its compatibility with the 
+following Delta Lake implementations:
+
+- Delta Lake OSS
+- Delta Lake Databricks
+
+
+## Delta Lake OSS Product tests
+
+Testing against Delta Lake OSS is quite straightforward by simply spinning up
+the corresponding product test environment:
+
+```
+testing/bin/ptl env up --environment 'EnvSinglenodeDeltaLakeOss'
+```
+
+
+## Delta Lake Databricks Product tests
+
+The Databricks Delta Lake implementation provides a richer functionality compared
+to the open source version.
+In order to setup a Databricks testing environment there are a few steps to be performed.
+At the time of this writing, Databricks is available on the following cloud providers:
+
+- AWS
+- Microsoft Azure
+- Google Cloud
+
+### Delta Lake Databricks on AWS
+
+Start by setting up a Databricks account via https://databricks.com/try-databricks and after
+filling your contact details, choose *AWS* as preferred cloud provider.
+
+Create an AWS S3 bucket to be used for storing the content of the Delta Lake tables managed
+by the Databricks runtime.
+
+Follow the guideline [Secure access to S3 buckets using instance profiles](https://docs.databricks.com/administration-guide/cloud-configurations/aws/instance-profiles.html)
+for allowing the Databricks cluster to access the AWS S3 bucket on which the Delta Lake tables are stored or AWS Glue
+table metastore.
+
+In order to make sure that the setup has been done correctly, proceed via Databricks Web UI to
+create a notebook on which a simple table could be created:
+
+```
+%sql
+CREATE TABLE default.test1 ( 
+    a_bigint BIGINT) 
+USING DELTA LOCATION 's3://my-s3-bucket/test1'
+```
+
+(Be sure to replace in the statement above `my-s3-bucket` with the name of your S3 bucket on which 
+the Databricks cluster has the ability to write to by the previously configured Instance profile).
```

Comment:
remove, don't state the obvious.
this is "Developer Notes", so you can be less explicit than in user's docs.